agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/tmds/handlers/v1/credentials_handler.go

```diff
@@ -116,23 +116,25 @@ func processCredentialsRequest(
 	credentials, ok := credentialsManager.GetTaskCredentials(credentialsID)
 	if !ok {
 		errText := errPrefix + "Credentials not found"
-		seelog.Errorf("Error processing credential request: %s", errText)
+		logger.Error(fmt.Sprintf("Error processing credential request: %s", errText))
 		msg := &handlersutils.ErrorMessage{
 			Code:          ErrInvalidIDInRequest,
 			Message:       errText,
 			HTTPErrorCode: http.StatusBadRequest,
 		}
 		return nil, "", "", msg, errors.New(errText)
 	}
-
-	seelog.Infof("Processing credential request, credentialType=%s taskARN=%s credentialScope=%s",
-		credentials.IAMRoleCredentials.RoleType, credentials.ARN, credentials.IAMRoleCredentials.CredentialScope)
+	logger.Info("Processing credential request", logger.Fields{
+		"credentialType":  credentials.IAMRoleCredentials.RoleType,
+		"taskARN":         credentials.ARN,
+		"credentialScope": credentials.IAMRoleCredentials.CredentialScope,
+	})
 
 	if utils.ZeroOrNil(credentials.ARN) && utils.ZeroOrNil(credentials.IAMRoleCredentials) {
 		// This can happen when the agent is restarted and is reconciling its state.
 		errText := errPrefix + "Credentials uninitialized for ID"
-		seelog.Errorf("Error processing credential request credentialType=%s taskARN=%s credentialScope=%s: %s",
-			credentials.IAMRoleCredentials.RoleType, credentials.ARN, credentials.IAMRoleCredentials.CredentialScope, errText)
+		logger.Error(fmt.Sprintf("Error processing credential request credentialType=%s taskARN=%s credentialScope=%s: %s",
```

Comment:
Thanks for making these changes!

In addition to replacing seelog with logger in this PR, we should refactor the message with fields defined [here](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/logger/field/constants.go) to make it easier to read. 

For example,
```
logger.Error("Error processing credential request", logger.Fields{
    "credentialType": credentials.IAMRoleCredentials.RoleType,
    field.TaskARN: credentials.ARN,
    "credentialScope": credentials.IAMRoleCredentials.CredentialScope,
    field.Error:                   errText,
})
```

Reply:
Also, we should think of if any cx application has dependency on the exact log message as these changes are made in TMDEv1.